### ISSUE TITLE
]f (evil-unimpaired/next-file) is not working

### ISSUE THREAD

**Issue by lislon** [syl20bnr/spacemacs#6437]
#### Description :octocat:
`]f` (evil-unimpaired/next-file) is not working. (Probably missing `f.el` library)

#### Reproduction guide :beetle:
- Start Emacs
- Open any file
- Type `]f`

*Observed behaviour:* :eyes: :broken_heart:
Error: `let*: Symbol's function definition is void: f-dirname`

*Expected behaviour:* :heart: :smile:
Navigate to next file in current directory.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.21
- Spacemacs branch: develop (rev. 1a13cba)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm emacs-lisp)
```

#### Backtrace :paw_prints:
```
Debugger entered--Lisp error: (void-function f-dirname)
  (f-dirname buffer-file-name)
  (let* ((directory (f-dirname buffer-file-name)) (files (f--files directory (not (s-matches\? "^\\.?#" it)))) (index (+ (-elem-index buffer-file-name files) offset)) (file (and (>= index 0) (nth index files)))) (if file (progn (f-expand file directory))))
  (progn (let* ((directory (f-dirname buffer-file-name)) (files (f--files directory (not (s-matches\? "^\\.?#" it)))) (index (+ (-elem-index buffer-file-name files) offset)) (file (and (>= index 0) (nth index files)))) (if file (progn (f-expand file directory)))))
  (if buffer-file-name (progn (let* ((directory (f-dirname buffer-file-name)) (files (f--files directory (not (s-matches\? "^\\.?#" it)))) (index (+ (-elem-index buffer-file-name files) offset)) (file (and (>= index 0) (nth index files)))) (if file (progn (f-expand file directory))))))
  evil-unimpaired//find-relative-filename(1)
  (let ((filename (evil-unimpaired//find-relative-filename 1))) (if filename (find-file filename) (user-error "No next file")))
  (-if-let* ((filename (evil-unimpaired//find-relative-filename 1))) (find-file filename) (user-error "No next file"))
  (-if-let (filename (evil-unimpaired//find-relative-filename 1)) (find-file filename) (user-error "No next file"))
  evil-unimpaired/next-file()
  call-interactively(evil-unimpaired/next-file nil nil)
  command-execute(evil-unimpaired/next-file)
```

**Comment by syl20bnr** [comment]
Should be fixed in develop.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
